sdk/identity/azure-identity/azure/identity/_credentials/environment.py

```diff
@@ -44,6 +44,9 @@ class EnvironmentCredential(object):
       - **AZURE_CLIENT_ID**: the service principal's client ID
       - **AZURE_CLIENT_CERTIFICATE_PATH**: path to a PEM or PKCS12 certificate file including the private key. The
         certificate must not be password-protected.
+      - **AZURE_CLIENT_SEND_CERTIFICATE_CHAIN**: Specifies whether an authentication request will include an x5c
+        header to support subject name / issuer based authentication.
+     * When set to `true` or `1`, authentication requests include the x5c header.
```

Comment:
Single backticks are used in some special Sphinx syntax. I don't know how they render when used this way. I think mundane quotes are good here but if you really want monospace, that's double backticks e.g. \``true\``